src/dirs.proj

```diff
@@ -44,7 +44,6 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <DisabledTestAssembly Include="System.Linq.Parallel.Tests" />
```

Comment:
Should we leave a commented item group so that it's easier for folks to disable tests in the future?